tests/notebooks/training-integration.ipynb

```diff
@@ -0,0 +1,904 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Name:\", paddlejob.metadata.name)\n",
+    "print(\"Spec:\", paddlejob.spec.paddle_replica_specs)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### List existing PaddleJobs\n",
+    "\n",
+    "List PaddleJobs in the current namespace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "[job.metadata.name for job in client.list_paddlejobs()]"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create PaddleJob\n",
+    "\n",
+    "Create a PaddleJob using the SDK."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client.create_paddlejob(paddlejob)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Get PaddleJob\n",
+    "Get the created PaddleJob by name and check its data.  \n",
+    "Make sure that it completes successfully before proceeding. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# verify that the Job was created successfully\n",
+    "# raises an error if it doesn't exist\n",
+    "paddlejob = client.get_paddlejob(name=PADDLEJOB_NAME)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# wait for the Job to complete successfully\n",
+    "assert_job_succeeded(client, PADDLEJOB_NAME, job_kind=\"PaddleJob\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Job:\", paddlejob.metadata.name, end=\"\\n\\n\")\n",
+    "print(\"Job Spec:\", paddlejob.spec, sep=\"\\n\", end=\"\\n\\n\")\n",
+    "print(\"Job Status:\", paddlejob.status, sep=\"\\n\", end=\"\\n\\n\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get PaddleJob Training logs\n",
+    "Get and print the training logs of the PaddleJob with the training steps "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print_training_logs(client, PADDLEJOB_NAME, container=\"paddle\", is_master=False)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete PaddleJob\n",
+    "\n",
+    "Delete the created PaddleJob and check that all created resources were removed as well."
```

Comment:
Same as above, we don't check for other created resources apart from the CR itself.